containers/nexus/configuration.sh

```diff
@@ -4,22 +4,15 @@ FQDN=$(get_fqdn)
 ADMUSR="$1"
 ADMPW="$2"
 ADMINGROUP="$3"
-tries=0
-while ! [ $(curl -sL -w "%{http_code}" "http://localhost:8081/nexus" -u $ADMUSR:$ADMPW -o /dev/null) -eq 200 ]
-do
-  ((tries++))
-  echo "wait for nexus (configuration)"
-  sleep 1
-  if [ $tries -gt 200 ]
-    then
-      echo "nexus didnt start"
-      echo "exit now"
-      exit 1
-  fi
-done
+
+if ! doguctl wait --port 8081 --timeout 120; then 
+  echo "Nexus seems not to be started. Exiting."
+  exit 1
+fi
+
 # set ADMINGROUP as nx-admins
 echo "set group $ADMINGROUP as nx-admins"
-curl -s -H "Content-Type: application/json" -X POST -d "{"data":{"id":"$ADMINGROUP","name":"$ADMINGROUP","description":"$ADMINGROUP","sessionTimeout":60,"roles":["nx-admin"],"privileges":[]}}" "http://localhost:8081/nexus/service/local/roles" -u $ADMUSR:$ADMPW
+curl --retry 3 --retry-delay 10 -s -H "Content-Type: application/json" -X POST -d "{data:{id:\"$ADMINGROUP\",name:\"$ADMINGROUP\",description:\"$ADMINGROUP\",sessionTimeout:60,roles:[nx-admin],privileges:[]}}" "http://localhost:8081/nexus/service/local/roles" -u "$ADMUSR":"$ADMPW"
```

Comment:
please use -s or --silent for curl commands, because the curl output clutters the log.